tests/TestCli.cpp

```diff
@@ -538,6 +543,22 @@ void TestCli::testClip()
     clipCmd.execute({"clip", m_dbFile2->fileName(), "--totp", "/Sample Entry"});
     m_stderrFile->seek(posErr);
     QCOMPARE(m_stderrFile->readAll(), QByteArray("Entry with path /Sample Entry has no TOTP set up.\n"));
+
+    posErr = m_stderrFile->pos();
+    Utils::Test::setNextPassword("a");
+    clipCmd.execute({"clip", m_dbFile->fileName(), "-a", "TESTAttribute1", "/Sample Entry"});
+    m_stderrFile->seek(posErr);
+    QCOMPARE(
+        m_stderrFile->readAll(),
+        QByteArray("ERROR: attribute TESTAttribute1 is ambiguous, it matches TestAttribute1 and testattribute1.\n"));
+
+    // Test handling of both TOTP and --attribute. Currently, this will cause
+    // --attribute to be ignored.
```

Comment:
@sjamesr Isn't it better to raise in that case? I feel like that should be an invalid invocation

Reply:
Done, please take a look.